l10n_es_facturae_auto_date/models/account_move.py

```diff
@@ -41,11 +41,10 @@ def create(self, vals):
             facturae = vals.get("facturae", partner.facturae)
             auto_date = vals.get("facturae_auto_dates", partner.facturae_auto_dates)
             if facturae and auto_date:
-                date_invoice = vals.get("date_invoice")
+                date_invoice = vals.get("invoice_date")
                 if date_invoice:
-                    if (
-                        vals.get("facturae_start_date") != False
-                        and vals.get("facturae_end_date") != False
```

Comment:
@MiquelRForgeFlow can you put those two lines back? They are necessary to distinguish between existing keys with False value and non-existing keys. Thank you

Reply:
done